frontend/src/components/MeetingRoom/LayoutButton/LayoutButton.tsx

```diff
@@ -4,26 +4,29 @@ import WindowIcon from '@mui/icons-material/Window';
 import { ReactElement } from 'react';
 import useSessionContext from '../../../hooks/useSessionContext';
 import ToolbarButton from '../ToolbarButton';
-import useIsSmallViewport from '../../../hooks/useIsSmallViewport';
 
 export type LayoutButtonProps = {
   isScreenSharePresent: boolean;
   isPinningPresent: boolean;
+  isOverflowButton?: boolean;
 };
 
 /**
  * LayoutButton Component
  *
  * Displays a button to toggle the meeting room layout for the user between `grid` and `active-speaker`.
  * @param {LayoutButtonProps} props - the props for the component.
+ *  @property {LayoutButtonProps} props - the props for the component
```

Comment:
we don't need to add `@property {LayoutButtonProps} props - the props for the component` here since we already have the `@param`